worlds/ror2/__init__.py

```diff
@@ -110,7 +110,8 @@ def fill_slot_data(self):
             "seed": "".join(self.world.slot_seeds[self.player].choice(string.digits) for i in range(16)),
             "totalLocations": self.world.total_locations[self.player].value,
             "totalRevivals": self.world.total_revivals[self.player].value,
-            "startWithDio": self.world.start_with_revive[self.player].value
+            "startWithDio": self.world.start_with_revive[self.player].value,
+            "FinalStageDeath": self.world.final_stage_death[self.player].value
```

Comment:
lmao I hate that this is inconsistent with the others but I know this is a client side thing.

Reply:
Oh, the capitalisation? My bad, I missed that. I can change it if you'd like.